packages/devtools_app/lib/src/vm_service_wrapper.dart

```diff
@@ -432,25 +432,38 @@ class VmServiceWrapper implements VmService {
   // https://github.com/dart-lang/sdk/blob/master/pkg/vm_service/lib/src/dart_io_extensions.dart
   Future<bool> isHttpTimelineLoggingAvailable(String isolateId) async {
     final Isolate isolate = await getIsolate(isolateId);
-    return isolate.extensionRPCs
-        .contains('ext.dart.io.setHttpEnableTimelineLogging');
-  }
-
-  Future<HttpTimelineLoggingState> getHttpEnableTimelineLogging(
-    String isolateId,
-  ) async {
-    assert(await isHttpTimelineLoggingAvailable(isolateId));
-    return _trackFuture('getHttpEnableTimelineLogging',
-        _vmService.getHttpEnableTimelineLogging(isolateId));
+    if (await isProtocolVersionSupported(
+        supportedVersion: SemanticVersion(major: 5, minor: 2))) {
```

Comment:
I don't think this is the right check. You want to check that the [service protocol extension version](https://github.com/dart-lang/sdk/blob/master/runtime/vm/service/service_extension.md) is `>= 1.2`, but it looks like you're comparing against the package revision instead.